torch/csrc/jit/export.cpp

```diff
@@ -195,12 +195,19 @@ void encodeGraph(onnx::GraphProto * p_g, const std::shared_ptr<Graph> & g, const
       // to help symbolics do the right thing.
       continue;
     }
+    if (node->kind() == stringToSymbol("OnnxElidedInput")) {
+      continue;
+    }
     auto p_n = p_g->add_node();
     if (node->getSourceLocation()) {
       p_n->set_doc_string(node->getSourceLocation()->python_traceback);
     }
     for(auto input : node->inputs()) {
-      p_n->add_input(value_name(input));
+      if (input->node()->kind() == stringToSymbol("OnnxElidedInput")) {
+        p_n->add_input("");
```

Comment:
Are we going to export input with empty name? We should have a better story about the optional parameter lists.

Reply:
Just talked to other developers, empty string is the current solution for the optional parameter list. So I am not gonna block this PR for this.